src/openfermion/utils/_givens_rotations.py

```diff
@@ -0,0 +1,190 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+"""Givens rotation decomposition of matrices."""
+from __future__ import absolute_import
+
+import numpy
+
+from openfermion.config import *
```

Comment:
usually it's a bad idea to import * because it pollutes the namespace in ways that aren't clear unless you actually look in the file. Instead, just import explicitly EQ_TOLERANCE, which is I think all you need.